crates/runtime/src/lib.rs

```diff
@@ -102,6 +102,7 @@ pub mod tls;
 pub mod tools;
 pub(crate) mod tracers;
 mod tracing_util;
+mod user_agent_util;
```

Comment:
We should have a `util` module with `tracing` and `user_agent` sub-modules rather than multiple top-level ones.